### PR TITLE
Adding GlusterFS as OCC

### PR DESCRIPTION
**Pull request by n0vabyte**

### CODE REVIEW

File: roles/post/templates/motd.j2
Comment (josephcardillo-akamai) [inline roles/post/templates/motd.j2:1]: Documentation link points to jitsi.

File: scripts/glusterfs-deploy.sh
Comment (josephcardillo-akamai) [inline scripts/glusterfs-deploy.sh:1]: The `cleanup` script isn't being called. /tmp/linode still exists on gluster1 after deployed.

Comment (josephcardillo-akamai) [reply scripts/glusterfs-deploy.sh:1]: The client IPs are still showing as required:
Screenshot 2024-04-15 at 2 26 09 PM

Comment (josephcardillo-akamai) [reply scripts/glusterfs-deploy.sh:1]: Is /tmp/linode still supposed to exist on gluster1 after deployment?

Comment (n0vabyte) [reply scripts/glusterfs-deploy.sh:1]: This should be fixed now.

**Pull request merged**